python_modules/dagster/dagster/components/lib/executable_component/component.py

```diff
@@ -63,6 +64,12 @@ def get_resources_from_callable(func: Callable) -> list[str]:
     return [param.name for param in sig.parameters.values() if param.name != "context"]
 
 
+# TODO: change to ResolvedAssetCheckSpec once alex/roach figure it out
```

Comment:
@alangenfeld @nicklausroach can wait for ResolveAssetCheckSpec if it will land today

Reply:
as far as i know https://github.com/dagster-io/dagster/pull/30184 fixed the only issue we had with `ResolvedAssetCheckSpec` - DOP just had not pulled in the update with that change yet

Reply:
yup. updated and restacked 👍🏻